app/main/index.js

```diff
@@ -358,6 +358,19 @@ app.on('ready', () => {
 	ipcMain.on('save-last-tab', (event, index) => {
 		ConfigUtil.setConfigItem('lastActiveTab', index);
 	});
+
+	// Update user idle status for each realm
+	// Set user idle if no activity for 60s
+	ipcMain.on('detect-presence', event => {
+		const idleThreshold = 60;
```

Comment:
Just to confirm, that number is definitely in seconds, not milliseconds or some other unit?  

(It might make sense to just have the comment mention the units rather than repeating the number, in case we change the number.).

Reply:
Yes, it is in seconds! Will add a comment for it!